plugins/json-extractor/src/parser.ts

```diff
@@ -13,52 +13,27 @@ export function parseBuffer(buffer: Buffer): WorkbookCapture {
     const parsedData = JSON.parse(fileContents) || []
     const results = Array.isArray(parsedData) ? parsedData : [parsedData]
 
-    // Ensure all items are objects
-    const filteredResults = results.filter(
-      (item) => typeof item === 'object' && item !== null
-    )
+    var sheetCapture = parseSheet(results);
 
-    if (filteredResults.length === 0) {
+    if (isEmpty(sheetCapture)) {
       return {} as WorkbookCapture
     }
 
-    // Custom flatten function
-    const flattenObject = (obj: any, parent: string = '', res: any = {}) => {
-      for (let key in obj) {
-        const propName = parent ? parent + '.' + key : key
-        if (typeof obj[key] === 'object') {
-          flattenObject(obj[key], propName, res)
-        } else {
-          res[propName] = obj[key]
-        }
-      }
-      return res
-    }
-
-    // Flatten the first item to determine headers
-    const firstItem = flattenObject(filteredResults[0])
-    const headers = Object.keys(firstItem).filter((header) => header !== '')
-
-    // Flatten and filter all rows
-    const filteredData = filteredResults.map((row) => {
-      const flattedRow = flattenObject(row)
-      return headers.reduce((filteredRow, header) => {
-        const cell = flattedRow[header]
-        filteredRow[header] = {
-          value: Array.isArray(cell) ? JSON.stringify(cell) : cell,
-        }
-        return filteredRow
-      }, {})
-    })
-
     return {
-      [sheetName]: {
-        headers,
-        data: filteredData,
-      },
+      [sheetName]: sheetCapture,
     } as WorkbookCapture
   } catch (error) {
```

Comment:
Bot suggestion: https://github.com/FlatFilers/flatfile-plugins/pull/693/files#r1826003535

Reply:
Ignoring for now